examples/react/contexts/WalletSelectorContext.tsx

```diff
@@ -71,6 +72,7 @@ export const WalletSelectorContextProvider: React.FC<{
         setupMathWallet(),
         setupNightly(),
         setupMeteorWallet(),
+        setupMetamaskSnap({}),
```

Comment:
It shouldn't be required for empty object to be present. You can have a look on other wallets/packages.